tests/Test.ADAL.NET.Unit.net45/InstanceDiscoveryTests.cs

```diff
@@ -173,7 +173,7 @@ public async Task TestInstanceDiscovery_WhenAuthorityIsAdfs_ShouldNotDoInstanceD
             using (var httpManager = new MockHttpManager())
             {
                 var serviceBundle = ServiceBundle.CreateWithCustomHttpManager(httpManager);
-                var authenticator = new Authenticator(serviceBundle, "https://login.contoso.com/adfs", false);
```

Comment:
>https://login.contoso.com/adfs [](start = 70, length = 30)

i believe this should still be possible

Reply:
yes it is. i removed the tenantId. 

---
In reply to: [378037207](https://github.com/AzureAD/azure-activedirectory-library-for-dotnet/pull/1694#discussion_r378037207) [](ancestors = 378037207)